docs/providers/aws/events/alb.md

```diff
@@ -103,3 +103,27 @@ module.exports.hello = async (event, context, callback) => {
 ```
 
 The handler response object must use `multiValueHeaders` to set HTTP response headers, `headers` would be ignored.
+
+## Prepending a prefix to generated target group names
+
+By default, target group names are strings generated by hashing a combined string of the function name, alb's id and whether multi-value headers are enabled. This produced a fixed length, unique id for target groups.
+
+Set the `providers.alb.targetGroupPrefix` attribute to a string to prepend this string to all target group names so that they become semi-predictable. The overall length is still truncated to 32 because this is a hard limit for target group names on AWS.
```

Comment:
_If you need target group names to have common predictable prefix, you may preset the prefix with `providers.alb.targetGroupPrefix` setting.  Note maximum length of the prefix is 16 chars_

Reply:
Updated with minor changes